### PR TITLE
Handle actor config + IT for actor state.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Handle actor config + IT for actor state.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #34

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [X] Created/updated tests
* [X] Extended the documentation

### CODE REVIEW

File: sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntimeConfig.java [hunk 1]
```diff
@@ -22,12 +45,92 @@ ActorRuntimeConfig addRegisteredActorType(String actorTypeName) {
...
+   * @param drainOngoingCallTimeout The timeout to drain ongoing calls.
+   * @return This instance.
+   */
+  public ActorRuntimeConfig setDrainOngoingCallTimeout(Duration drainOngoingCallTimeout) {
+    this.drainOngoingCallTimeout = drainOngoingCallTimeout;
+    return this;
+  }
+
+  /**
+   * Gets whether balances actors should be drained.
```
Comment (LMWF) [inline sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntimeConfig.java]: Balanced

File: sdk-tests/src/test/java/io/dapr/it/actors/ActorStateIT.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      proxy.invokeActorMethod("writeMessage", message).block();
+    }, 5000);
+
+    callWithRetry(() -> {
+      logger.debug("Invoking readMessage where data is probably still cached ... ");
+      String result = proxy.invokeActorMethod("readMessage", String.class).block();
+      assertEquals(message, result);
+    }, 5000);
+
+    StatefulActor.MyData mydata = new StatefulActor.MyData();
```
Comment (LMWF) [inline sdk-tests/src/test/java/io/dapr/it/actors/ActorStateIT.java]: You may want to put a log here explaining the different between this and above.  From my reading of this, the difference is between a String and a user-defined object.

File: sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntimeConfig.java (hunk 1 above)
Comment (artursouza) [reply sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntimeConfig.java]: Done.

File: sdk-tests/src/test/java/io/dapr/it/actors/ActorStateIT.java (hunk 2 above)
Comment (artursouza) [reply sdk-tests/src/test/java/io/dapr/it/actors/ActorStateIT.java]: Done.

**Pull request merged**